### ISSUE TITLE
Avenger (xcom row perk) is triggering on RNF

### ISSUE THREAD

**Issue by illogicalproductions** [long-war-2/lwotc#1500]
Avenger, which should trigger on a shot onto an ally, is triggering onto RNF pods as they land.

**Comment by pledbrook** [comment]
Save file that reproduces the problem?

**Comment by pledbrook** [comment]
Also , what do you mean exactly by "triggering onto RNF pods as they land"? The unit with Avenger is shooting at the pod members when they drop?

**Comment by illogicalproductions** [comment]
Yes. When RNF comes off drop-ship, avenger is triggering onto them.  As far as save, I will try and get one this week.

**Comment by copyrite** [comment]
Should be decided if this is indeed a bug, or whether the hostile action (in this case dropping as reinforcement) without a specific target is an undocumented feature.
